app/src/main/java/org/mozilla/fenix/utils/ColorUtils.kt

```diff
@@ -0,0 +1,3 @@
+package org.mozilla.fenix.utils
```

Comment:
Please run our linters detekt and ktlint locally: `./gradlew detekt ktlint`

Couple of comments here:
1. Let's call this the `Color.kt` file - we try to avoid the "utils" naming convention to avoid creating dumping grounds.
2. There is a missing license that should be at the top of this file.
3. Add a KDoc that should be on this extension function.
4. `toHexColorLong` -> `toHexColor` - the type is inferred.

Reply:
Please squash them to make all the commits into one commit. Thanks!